mdn/developer-portal
# 1229 Social icons update

This changeset refreshes the look of the social icons without breaking the CSS of any other icons which are still in use (eg the close X triggers in the filter list)

The social icon SVGs have been made to use a smaller viewbox and the CSS that styles them in the `.icon` class has been reinvented in a more focused class (a child of `.resource share` to avoid contaminating other places that use `.icon`)

It also fixes an existing bug where the social links were linking to the site, but not to the specific page which the social links are on.

Cross-browser tested and fine everywhere, even IE11 😄 

----

![Screenshot 2020-04-03 at 15 07 01](https://user-images.githubusercontent.com/101457/78369989-ed604700-75bd-11ea-8128-047b3522a245.png)

----

![Screenshot 2020-04-03 at 14 24 20](https://user-images.githubusercontent.com/101457/78369991-ee917400-75bd-11ea-8d94-bb12ddea854e.png)

----

(Related issue #1229)

## How to test

- Code is up on dev. 
- Please check a Post, a Video and an Event (which are the three places the new-style links are used)
- Please confirm that the filter list on the Posts, People or Events pages shows the expected "X" Icon to clear selections when some filters have been applied
